plugins/kotlin/j2k/shared/tests/testData/newJ2k/assignmentExpression/expressionAssignmentWithParentheses.kt

```diff
@@ -7,10 +7,10 @@ internal object Test {
         var d = true
         val e = true
 
-        if (e.let { d = d and it; d }.also { (c) = it }.let { b = b or it; b }.also { a = it });
+        if (e.let { d = d and it; d }.also { c = it }.let { b = b or it; b }.also { a = it });
         while ((((((b.also { a = it }))))));
         do {
-        } while (((b.let { (a) = (a) xor it; (a) })))
+        } while (((b.let { a = (a) xor it; (a) })))
```

Comment:
I know it is not strictly related, but is it possible to clean up this mess? It seems like it should not be hard to drop straight nested parentheses.

Reply:
Oh, sorry, it was in the original file, and we just preserve the parentheses, right? Then no problem here.